<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Input File :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputLineEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="chooseInputButton">
            <property name="text">
             <string>Choose File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Output File :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="outputLineEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="chooseOutputButton">
            <property name="text">
             <string>Choose File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Regex Research : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="regexLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkCase">
          <property name="text">
           <string>Case Sensitive</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkOrder">
          <property name="text">
           <string>Alphabetical/Frequency Order</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkAll">
          <property name="text">
           <string>All Statistics</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkLetters">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Letters Statistics</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkWords">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Words Statistics</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkDouble">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Double Letters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTriple">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Triple Letters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkCSV">
          <property name="text">
           <string>CSV Statistics</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkRegex">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Regex Statistics</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkPie">
          <property name="text">
           <string>Pie Charts Visual</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkCharts">
          <property name="text">
           <string>Charts Visual</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkHeatmap">
          <property name="text">
           <string>HeatMap Visual</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lanchButton">
          <property name="text">
           <string>Lanch Research</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="resultPlainText"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
